services/cvm.js

```diff
@@ -0,0 +1,92 @@
+const AdmZip = require('adm-zip');
+const request = require('request');
+
+function requestCvmFile() {
+  const file_url =
+    'http://dados.cvm.gov.br/dados/INTERMED/CAD/DADOS/cad_intermed.zip';
+  return new Promise(function (resolve, reject) {
+    request.get({ url: file_url, encoding: null }, function (
+      error,
+      response,
+      body
+    ) {
+      if (error) return reject(error);
+      try {
+        resolve(body);
+      } catch (e) {
+        reject(e);
+      }
+    });
+  });
+}
+
+export const getExchangesData = async () => {
+  const LINE_BREAK = '\r\n';
+
+  return await requestCvmFile()
```

Comment:
Estando em um contexto `async` podemos evitar o uso de Promises (`.then` e o `.catch`), podendo ficar assim:

```suggestion
const body = await requestCvmFile()
const zip = new AdmZip(body);
```